contrib/automation_tests/orbit_stadia_symbol_store.py

```diff
@@ -33,6 +33,7 @@
  - (post clean up) disable stadia symbol store
 """
 
+
```

Comment:
I dont know if this is on purpose. I guess it is?

Reply:
Seems to be python style suggestion, like two empty lines above functions...

Reply:
Yeah, just a linter suggestion a bumped into.